bgpd/bgp_pbr.c

```diff
@@ -1817,8 +1819,21 @@ static void bgp_pbr_policyroute_add_to_zebra_unit(struct bgp *bgp,
 		bpme->install_in_progress = false;
 		/* link bgp info to bpme */
 		bpme->bgp_info = (void *)binfo;
-	}
+	} else
+		bpme_found = true;
```

Comment:
what is bpme_found?  You can get in here without a valid bpme pointer.  I don't understand the name

Reply:
This code can be simplified, since hash_get obviously always returns a context.
bpme_found means that an entry in FS PBR already is present ( because the unique field is already initialised); that means that the entry tries to be installed twice, which should be prevented.